vertica/datadog_checks/vertica/views.py

```diff
@@ -68,33 +68,65 @@ class Projections:
     query = 'SELECT {} FROM v_catalog.{}'.format(', '.join(fields), name)
 
 
-class ProjectionStorage:
-    """
-    https://www.vertica.com/docs/9.2.x/HTML/Content/Authoring/SQLReferenceManual/SystemTables/MONITOR/PROJECTION_STORAGE.htm
-    """
-
-    name = 'projection_storage'
-    fields = (
-        'anchor_table_name',
-        'node_name',
-        'projection_name',
+def make_projection_storage_queries(version):
```

Comment:
Did you look into keeping these classes but generating the queries differently? I think it's a little confusing to have functions for these two types of queries but classes for `Nodes`, `Projections`, etc

Reply:
I couldn't see any way to keep things homogeneous that didn't involve writing individual classes for each query + version-divide combination, but that would have been more code and repetition for little gain. The classes, as they are, are little more than namespaces, so it's unclear that there's any benefit to sticking to that only for code homogeneity. Plus, I intend to refactor the integration to make use of the `QueryManager` which I think could involve making all those classes disappear.

Reply:
Ah, sounds good. I'm fine with keeping this approach if you plan to do a QueryManager refactor in the future!